common/logger/src/lib.rs

```diff
@@ -87,69 +47,22 @@
 //!     #[schema(display)]
 //!     c: Option<&'a str>,
 //! }
```

Comment:
Maybe mention debug as well.

Reply:
It is mentioned, just github didn't do a good job showing it :)